src/hooks/useFilter.ts

```diff
@@ -1,48 +1,64 @@
-import { toLower, isString, isBoolean, isNumber } from "lodash";
-import { Offer } from "src/types/offer";
+import { isBoolean, isNumber, isString, toLower } from 'lodash';
+
+import { Offer } from 'src/types/offer';
 
 const skipedKey = [
-    "offerTokenAddress",
-    "offerTokenDecimals",
-    "buyerTokenAddress",
-    "buyerTokenDecimals",
-    "createdAtTimestamp",
-    "buyCurrency"
+  'offerTokenAddress',
+  'offerTokenDecimals',
+  'buyerTokenAddress',
+  'buyerTokenDecimals',
+  'createdAtTimestamp',
+  'buyCurrency',
 ];
 
 const filterDatas = (filterValue: string, datas: Offer[]) => {
-    if(!filterValue) return datas;
+  if (!filterValue) return datas;
 
-    return datas.filter((data: Offer) => {
-        return Object.keys(data).map(key => {
-            if(skipedKey.includes(key)) return false;
+  return datas.filter((data: Offer) => {
+    return Object.keys(data)
+      .map((key) => {
+        if (skipedKey.includes(key)) return false;
 
-            const value = data[key as keyof typeof data];
+        const value = data[key as keyof typeof data];
 
-            if(isString(value)){
-                return toLower(value).includes(toLower(filterValue))
-            }
+        if (isString(value)) {
+          return toLower(value).includes(toLower(filterValue));
+        }
 
-            if (isBoolean(value)) {
-                return (filterValue === 'true' && value) || (filterValue === 'false' && !value)
-            }
+        if (isBoolean(value)) {
+          return (
+            (filterValue === 'true' && value) ||
+            (filterValue === 'false' && !value)
+          );
+        }
 
-            if (isNumber(value)) {
-                return value.toString() == filterValue
-            }
+        if (isNumber(value)) {
+          return value.toString() == filterValue;
+        }
 
-            return false
-        }).includes(true);
-    });
-}
+        return false;
+      })
+      .includes(true);
+  });
+};
 
-interface UseFilter{
-    filteredDatas: Offer[]
+interface UseFilter {
+  filteredDatas: Offer[];
 }
 
 export const useFilter = (filterValue: string, datas: Offer[]): UseFilter => {
+  return {
+    filteredDatas: filterDatas(filterValue, datas),
+  };
+};
```

Comment:
Be careful when you commit changes that have nothing to do with the feature you are pushing. The good practice in this case you wan to keep your style change is to have a separate commit that will look something like this: `style(useFilter): cleaner indentation`. [See here for more info on conventional commit](https://www.conventionalcommits.org/en/v1.0.0/)

Reply:
You are right, really complicated for me to find which code is really modified

Reply:
I removed all the style that had nothing to do with the feature in PR #34

Reply:
Michael told me that the feature is not working properly

Reply:
@NandyBa probably had to code the feature without the UI. Is there anyway we can get the NEXT_PUBLIC_API_URL ?

Reply:
Even if i give you this env var you will not be able to use the app because you need an auth token and we can't give token right now.

Reply:
@NandyBa @0xEzSwim You can check the working solution here:
https://github.com/real-token/realtoken-yam-interface/commit/0a46810b1593aba9c4abc50adf17378bc8bd1461